src/tabs/tab-header-bar.scss

```diff
@@ -71,27 +71,37 @@ $label-horizontal-spacing: awsui.$space-l;
   display: flex;
   max-inline-size: calc(90% - #{$label-horizontal-spacing});
 
-  & > button {
+  & > div,
+  & > div > button {
```

Comment:
Maybe it's time to give these elements proper class names so these styles are easier to read?